### PR TITLE
1.1 add roles on media folders

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/1163
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/62

### CODE REVIEW

File: MediaAdminBundle/Form/Type/OrchestraSiteForFolderChoiceType.php [hunk 1]
```diff
@@ -75,7 +75,7 @@ protected function getChoices()
         $userGroups = $this->tokenStorage->getToken()->getUser()->getGroups();
         /** @var GroupInterface $group */
         foreach ($userGroups as $group) {
-            if (in_array($group->getSite()->getSiteId(), $siteIds) && ($group->hasRole('ROLE_ACCESS_TREE_FOLDER'))) {
+            if (in_array($group->getSite()->getSiteId(), $siteIds) && ($group->hasRole('ROLE_ACCESS_MEDIA_FOLDER'))) {
```
Comment (alavieille) [inline MediaAdminBundle/Form/Type/OrchestraSiteForFolderChoiceType.php]: replace hasRole by method isGranted of authorizationChecker and
a group does not necessarily a site

**Comment by ngilain** [comment]
@itkg-alavieille : we can't use the isGranted method because we d'on't check the right on the current site but on a list of sites. But I have fixed the problem with groups having no site.

**Comment by alavieille** [comment]
:+1:

**Comment by ngilain** [comment]
@nicolasThal : fixtures moved

**Pull request merged**